projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/datajobs/webhook/DeploymentModelConverterV2.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2021-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.vmware.taurus.datajobs.webhook;
+
+import com.vmware.taurus.service.model.DataJobDeployment;
+import com.vmware.taurus.service.model.JobDeployment;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+/** Utility functions that convert between different model or DTO classes */
+@Service
+@AllArgsConstructor
+public class DeploymentModelConverterV2 {
+
+  public static DataJobDeployment toJobDeployment(
```

Comment:
According to the latest development approach, you should move this method to DeploymentModelConverter.